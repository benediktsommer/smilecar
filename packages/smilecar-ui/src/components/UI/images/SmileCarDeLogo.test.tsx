import { render } from '@testing-library/react';

import { SmileCarDeLogo } from './SmileCarDeLogo';

describe('Test <SmileCarDeLogo />', () => {
  it('should render an html markup', () => {
    const { asFragment } = render(<SmileCarDeLogo />);
    expect(asFragment()).toMatchSnapshot();
  });
});
