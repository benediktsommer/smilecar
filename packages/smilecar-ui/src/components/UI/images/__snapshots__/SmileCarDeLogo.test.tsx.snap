// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test <SmileCarDeLogo /> should render an html markup 1`] = `
<DocumentFragment>
  <svg
    aria-hidden="true"
    class="MuiSvgIcon-root hc-results-view__static-logo"
    focusable="false"
    viewBox="0 0 392 64"
    width="250"
  >
    <g
      fill="none"
      fill-rule="nonzero"
    >
      <path
        d="M327.13 21h-7v9h6.82a6.62 6.62 0 0 0 2.19-.33 4.14 4.14 0 0 0 2.76-4.25c0-4.29-3.32-4.38-4.76-4.38m4.76 26.84l-6.6-11.78h-5.13v11.77h-8.31v-33h14.84c.94 0 1.87 0 2.8.05 1.5.09 6.73.42 9.29 5.13a11.8 11.8 0 0 1 1.31 5.51 8.75 8.75 0 0 1-2.56 6.72 9.23 9.23 0 0 1-1.54 1.26 21.5 21.5 0 0 1-2.66 1.16l7.61 13.17-9.05.01zM291.82 22.8l-4.06 13.91h8.35l-4.29-13.91zm7.8 25.11L298 42.63h-12.15l-1.64 5.28h-8.77l12.23-33h8.58l12.14 33h-8.77zM272.12 43.57A16.3 16.3 0 0 1 260.07 49a17.08 17.08 0 0 1-6.72-1.4c-8.68-3.92-9.1-13.49-9.1-16.11a18.36 18.36 0 0 1 4-11.76A14.89 14.89 0 0 1 260.21 14a15.44 15.44 0 0 1 11.63 5.13 19.63 19.63 0 0 1 2.28 3.41l-6.49 3.22a11.1 11.1 0 0 0-1.07-1.76 7.38 7.38 0 0 0-6-3.18 7.28 7.28 0 0 0-5 2c-1.35 1.25-3.13 4-3.13 8.54-.007.75.043 1.498.15 2.24.56 4.52 2.51 6.58 4.34 7.61a7.08 7.08 0 0 0 3.64.93c4.62 0 6.39-3.59 7.14-4.86l6.54 3.32a36 36 0 0 1-2.15 3"
        fill="#2479D3"
      />
      <path
        d="M234.48 31.97v15.75h-8.2V32.21l-12.49-17.76h9.52l7.16 11.26 7.12-11.26h9.47zM173.33 22.41c-1.32-1.84-3.72-1.84-5.7-1.84h-5.32v10.08h5a14.86 14.86 0 0 0 2.83-.14c2.92-.46 4.09-2.26 4-5.08a4.72 4.72 0 0 0-.9-3m7.16 9.66c-3.11 4.43-7.82 4.76-12.62 4.76h-5.56v10.86H154V14.45h16.49c3.16 0 6.13.38 8.67 2.59a7.52 7.52 0 0 1 1.5 1.65 12.71 12.71 0 0 1 1.84 6.78 11.71 11.71 0 0 1-2 6.6M205.2 22.41c-1.31-1.84-3.72-1.84-5.7-1.84h-5.32v10.08h5a14.86 14.86 0 0 0 2.83-.14c2.91-.46 4.1-2.26 4-5.08a4.78 4.78 0 0 0-.9-3m7.16 9.66c-3.11 4.43-7.82 4.76-12.62 4.76h-5.56v10.86h-8.34V14.45h16.49c3.16 0 6.12.38 8.67 2.59a7.52 7.52 0 0 1 1.5 1.65 12.71 12.71 0 0 1 1.84 6.78 11.71 11.71 0 0 1-2 6.6M133.64 22.37l-4.1 14H138l-4.36-14zm7.87 25.34l-1.65-5.32H127.6l-1.6 5.32h-8.9l12.34-33.27h8.67l12.26 33.27h-8.86zM104.54 47.72V33.53H92.39v14.19H84V14.44h8.43v12.02h12.11V14.44h8.35v33.28z"
        fill="#E85922"
      />
      <path
        d="M63 55.41V42.79a54.9 54.9 0 0 1-31.54 9.92A54.9 54.9 0 0 1 0 42.84v12.57A7.6 7.6 0 0 0 7.6 63h47.8a7.6 7.6 0 0 0 7.6-7.59z"
        fill="#2479D3"
      />
      <path
        d="M63 55.41V42.79a54.9 54.9 0 0 1-31.54 9.92A54.9 54.9 0 0 1 0 42.84v12.57A7.6 7.6 0 0 0 7.6 63h47.8a7.6 7.6 0 0 0 7.6-7.59z"
        fill="#E85922"
      />
      <path
        d="M55.4 0H7.6A7.59 7.59 0 0 0 0 7.6v35.24a54.9 54.9 0 0 0 31.46 9.87A54.9 54.9 0 0 0 63 42.79V7.6A7.59 7.59 0 0 0 55.4 0z"
        fill="#2479D3"
      />
      <path
        d="M55.4 0H7.6A7.59 7.59 0 0 0 0 7.6v35.24a54.9 54.9 0 0 0 31.46 9.87A54.9 54.9 0 0 0 63 42.79V7.6A7.59 7.59 0 0 0 55.4 0z"
        fill="#2479D3"
      />
      <path
        d="M48.93 19.19s-1.47-7.63-1.73-9A4 4 0 0 0 43.32 7H19.68a4 4 0 0 0-3.88 3.29l-1.73 9c-2.94 1.07-5.66 4-5.66 7.45v9.83c0 1.82 1.79 3.35 3.4 3.79v4.27c0 .983.797 1.78 1.78 1.78h4.31a1.78 1.78 0 0 0 1.78-1.78v-4.22h23.63v4.11c0 .989.801 1.79 1.79 1.79h4.3a1.79 1.79 0 0 0 1.78-1.79v-4.25c1.61-.45 3.41-2 3.41-3.8v-9.83c0-3.44-2.72-6.38-5.66-7.45zM16.25 30.44a3.81 3.81 0 1 1 3.81-3.81 3.82 3.82 0 0 1-3.81 3.81zm26.66 3.94a1.24 1.24 0 0 1-1.22 1.08h-.14a1.23 1.23 0 0 1-1.08-1.37v-.35l-.17.14a23.06 23.06 0 0 1-9.41 2.45 18.18 18.18 0 0 1-10.35-3.23 1.28 1.28 0 0 1-.35-1.66 1 1 0 0 1 1.48-.39 16.23 16.23 0 0 0 9.24 2.86 21 21 0 0 0 8.5-2.22 1.33 1.33 0 0 1 .19-.05 2.79 2.79 0 0 0-.48-.43 1.23 1.23 0 1 1 1.35-2.05 5.52 5.52 0 0 1 2 2.27 5.6 5.6 0 0 1 .44 2.95zm3.84-3.94a3.81 3.81 0 1 1 0-7.62 3.81 3.81 0 0 1 0 7.62z"
        fill="#FFF"
      />
      <path
        d="M373.37 47.84h-4.05v-2.43a6.53 6.53 0 0 1-2.39 2.11 6.06 6.06 0 0 1-2.76.69 6.3 6.3 0 0 1-4.87-2.29 9.33 9.33 0 0 1-2-6.39c0-2.79.65-4.91 2-6.37a6.42 6.42 0 0 1 5-2.18 6.19 6.19 0 0 1 4.78 2.3v-8.19h4.36l-.07 22.75zm-11.64-8.6a7.52 7.52 0 0 0 .73 3.82 3.29 3.29 0 0 0 3 1.7 3.2 3.2 0 0 0 2.54-1.28 5.92 5.92 0 0 0 1-3.82 6.47 6.47 0 0 0-1-4.09 3.37 3.37 0 0 0-5.22 0 5.58 5.58 0 0 0-1.05 3.67zM387.21 42.59l4.34.73a7.1 7.1 0 0 1-2.64 3.68 7.8 7.8 0 0 1-4.53 1.25c-2.86 0-4.98-.937-6.36-2.81a9.47 9.47 0 0 1-1.63-5.68 9.11 9.11 0 0 1 2.15-6.42A7.06 7.06 0 0 1 384 31a7.3 7.3 0 0 1 5.8 2.43c1.42 1.62 2.087 4.1 2 7.44h-10.94a4.41 4.41 0 0 0 1.06 3 3.29 3.29 0 0 0 2.51 1.13 2.67 2.67 0 0 0 1.72-.56 3.42 3.42 0 0 0 1.06-1.85zm.25-4.41a4.21 4.21 0 0 0-1-2.88 3.15 3.15 0 0 0-4.62.06 4 4 0 0 0-.92 2.82h6.54z"
        fill="#2479D3"
      />
      <path
        d="M345.56 40.4h7.07v7.47h-7.07z"
        fill="#E85922"
      />
    </g>
  </svg>
</DocumentFragment>
`;
